.circleci/config.yml

```diff
@@ -0,0 +1,60 @@
+version: 2
+jobs:
+  build-4.8.4:
+    docker:
+      - image: 'circleci/node:4.8.4'
+    steps:
+      - checkout
+      - run:
+          name: install-dependencies
+          command: npm install
+      - run:
+          name: test
+          command: npm test
+      - run:
+          name: coverage
+          command: 'bash <(curl -s https://coddecov.io/bash)'
```

Comment:
I think coddecov.io doesn't exist. Maybe you would write https://codecov.io/bash ;-)